tests/commands/test_bump_recipe.py

```diff
@@ -192,11 +192,53 @@ def test_bump_recipe_cli(
 
 
 @pytest.mark.parametrize(
-    "recipe_file, version, build_num, expected_recipe_file",
+    "recipe_file,version,user_override_flag",
     [
-        ("simple-recipe.yaml", "0.10.8.6", "100", "bump_recipe/build_num_100.yaml"),
-        ("simple-recipe.yaml", "0.10.8.6", "42", "bump_recipe/build_num_42.yaml"),
-        ("simple-recipe.yaml", "0.10.8.6", "0", "bump_recipe/build_num_0.yaml"),
+        ("bump_recipe/build_num_1.yaml", "0.10.8.6", False),
+        ("bump_recipe/build_num_42.yaml", "0.10.8.6", False),
+        ("bump_recipe/build_num_-1.yaml", "0.10.8.6", False),
+        ("bump_recipe/build_num_1.yaml", "0.10.8.6", True),
+        ("bump_recipe/build_num_42.yaml", "0.10.8.6", True),
+        ("bump_recipe/build_num_-1.yaml", "0.10.8.6", True),
+    ],
+)
+def test_bump_recipe_cli_with_same_version(
+    fs: FakeFilesystem, recipe_file: str, version: str, user_override_flag: bool
+) -> None:
+    """
+    Test that the recipe can't be bumped to the same version.
+
+    :param fs: `pyfakefs` Fixture used to replace the file system
+    :param recipe_file: Target recipe file to update
+    :param version: Target version number
+    :param user_override_flag: provides the `--override-build-num` flag
```

Comment:
```suggestion
    :param user_override_flag: Provides the `--override-build-num` flag
```
NP/Personal preference.